recurrence/base.py

```diff
@@ -653,6 +651,16 @@ def __repr__(self):
     MO, TU, WE, TH, FR, SA, SU) = WEEKDAYS = list(map(lambda n: Weekday(n), range(7)))
 
 
+def to_utc(dt):
+    if not dt:
+        return dt
+
+    if is_naive(dt):
+        dt = make_aware(dt)
```

Comment:
Previously `localize` was used but `make_aware` should offer similar logic with Django timezone support and DST aware datetimes.